hw_wordsim.ipynb

```diff
@@ -1145,8 +1145,17 @@
   "widgets": {
    "state": {},
    "version": "1.1.2"
+  },
```

Comment:
Does this necessary for the PR?
I know it's tricky to update `ipynb` since it carries other changes, but if it is just from the running setup, then can this be cleaned?